src/ItemValue.ts

```diff
@@ -155,7 +155,7 @@ export class ItemValue implements IItemValue {
      * @param value   The desired value to now store.
      */
     public setValue(value: any): void {
-        if (this.transformSet) {
+        if (this.hasOwnProperty("transformSet")) {
```

Comment:
Just revert the change. The `.hasOwnProperty(foo)` vs `.foo` debate is one we shouldn't wade into, largely because I haven't made up a solidified opinion for FSS projects.